package.json

```diff
@@ -39,7 +42,23 @@
     "winston": "3.3.3"
   },
   "devDependencies": {
-    "speculate": "1.7.4"
+    "@babel/cli": "^7.8.4",
```

Comment:
There are newer versions of this package. Is there any reason why you put here older one? Also why package.lock.json is not commited?

Reply:
Package-lock is a good catch! Although I've been basing on most stable solution as I read but I can try to upgrade.